{{ .ProjectSnake }}/README.bazel.md

```diff
@@ -34,6 +34,24 @@ Similarly, you can run other `pnpm` commands to install or remove packages.
 This ensures you use the same pnpm version as other developers, and the lockfile format will stay constant.
 {{- end }}
 
+{{ if .Computed.python }}
+## Working with Python packages
+
+After adding a new `import` statement in Python code, run these commands:
```

Comment:
I think for beginners that is a little misleading, these instructions merge the flows for both 1p and 3p dependencies, where 1p dependencies require only the last step. Can this be made clearer?

Reply:
Yes, same for Go actually.